### PR TITLE
Move containers for sarek, demultiplex and scrnaseq to quay.io

### PR DESCRIPTION
**Pull request by drpatelh**
I have pulled and pushed the containers in these modules to the nf-core `quay.io` account. Updating modules now, will re-install in the pipeline and create patch releases.

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/gatk4/markduplicatesspark/main.nf [hunk 1]
```diff
@@ -3,7 +3,7 @@ process GATK4_MARKDUPLICATES_SPARK {
     label 'process_high'
 
     conda "bioconda::gatk4=4.3.0.0 conda-forge::openjdk=8.0.312"
-    container 'docker.io/broadinstitute/gatk:4.4.0.0'
+    container 'nf-core/gatk:4.4.0.0'
```
Comment (maxulysse) [inline modules/nf-core/gatk4/markduplicatesspark/main.nf:6]:
```suggestion
    container "nf-core/gatk:4.4.0.0"
```
we need double quotes

File: modules/nf-core/gatk4/applybqsrspark/main.nf [hunk 2]
```diff
@@ -3,7 +3,7 @@ process GATK4_APPLYBQSR_SPARK {
     label 'process_low'
 
     conda "bioconda::gatk4=4.3.0.0 conda-forge::openjdk=8.0.312"
-    container 'docker.io/broadinstitute/gatk:4.4.0.0'
+    container 'nf-core/gatk:4.4.0.0'
```
Comment (maxulysse) [inline modules/nf-core/gatk4/applybqsrspark/main.nf:6]:
```suggestion
    container "nf-core/gatk:4.4.0.0"
```

File: modules/nf-core/gatk4/baserecalibratorspark/main.nf [hunk 3]
```diff
@@ -3,7 +3,7 @@ process GATK4_BASERECALIBRATOR_SPARK {
     label 'process_low'
 
     conda "bioconda::gatk4=4.4.0.0 conda-forge::openjdk=8.0.312"
-    container 'docker.io/broadinstitute/gatk:4.4.0.0'
+    container 'nf-core/gatk:4.4.0.0'
```
Comment (maxulysse) [inline modules/nf-core/gatk4/baserecalibratorspark/main.nf:6]:
```suggestion
    container "nf-core/gatk:4.4.0.0"
```

**Comment by drpatelh** [comment]
All Docker tests are passing so will merge. We need to submit a separate issue to fix the linting with a list of modules failing here because it needs to be fixed in #tools.

**Pull request merged**